src/main/java/com/rackspace/ceres/app/web/LookupApiController.java

```diff
@@ -28,9 +40,11 @@ public LookupApiController(MetadataService metadataService) {
     }
 
     @GetMapping("/lookup")
+    @ApiOperation(value = "This api is used  to determine what time series are associated"
+        + " with a given metric, tag name, tag value, tag pair or combination thereof")
     public Mono<LookupResult> query(@RequestParam(name = "m") String m,
                                     @RequestParam(name = "limit", required = false) Integer limit,
-                                    @RequestHeader(value = "X-Tenant") String tenantHeader) {
+                                    @ApiIgnore @RequestHeader(value = "#{appProperties.tenantHeader}") String tenantHeader) {
```

Comment:
Can you move these ApiIgnore's to newlines?

Reply:
This ApiIgnore is with `@RequestHeader` that is why I add them together.